evm/deploy-its.js

```diff
@@ -44,12 +47,17 @@ async function deployAll(config, wallet, chain, options) {
     const verifyOptions = verify ? { env, chain: chain.axelarId, only: verify === 'only' } : null;
 
     const provider = getDefaultProvider(chain.rpc);
-    const InterchainTokenService = getContractJSON('InterchainTokenService', artifactPath);
 
     const contractName = 'InterchainTokenService';
     const itsFactoryContractName = 'InterchainTokenFactory';
     const contracts = chain.contracts;
 
+    const interchainTokenContractName = isHyperliquidChain(chain) ? 'HyperliquidInterchainToken' : 'InterchainToken';
+    const ServiceContract = getContractJSON(
```

Comment:
```suggestion
    const InterchainTokenService = getContractJSON(
```
this name is fine I think to still get the core idea across